src/Microsoft.AspNetCore.Mvc.Core/ApplicationParts/RelatedAssemblyAttribute.cs

```diff
@@ -117,6 +117,16 @@ internal static string GetAssemblyLocation(Assembly assembly)
         {
             if (Uri.TryCreate(assembly.CodeBase, UriKind.Absolute, out var result) && result.IsFile)
             {
+                if (!string.IsNullOrWhiteSpace(result.Fragment))
```

Comment:
I'd just do `Uri.TryCreate(assembly.CodeBase, UriKind.Absolute, out var result) && result.IsFile && string.IsNullOrEmpty(result.Fragment))`. Using `LocalPath` works well most of the times

Reply:
Really? ok =] Fixing